frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -30,6 +19,28 @@ export const getCephPVs = (pvsData: K8sResourceKind[] = []): K8sResourceKind[] =
     );
   });
 
+const getPVStorageClass = (pv: K8sResourceKind) => _.get(pv, 'spec.storageClassName');
+const getStorageClassName = (pvc: K8sResourceKind) =>
+  _.get(pvc, ['metadata', 'annotations', 'volume.beta.kubernetes.io/storage-class']) ||
+  _.get(pvc, 'spec.storageClassName');
```

Comment:
nit: We're tying to avoid `_.get` because it's not type safe. We use it all over, though, so I won't block on this :)